test/cri-containerd/stats_test.go

```diff
@@ -0,0 +1,152 @@
+// +build functional
+
+package cri_containerd
+
+import (
+	"context"
+	"testing"
+
+	runtime "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+)
+
+func Test_UVMStats_Single_LCOW(t *testing.T) {
```

Comment:
Can we call these sandbox stats since this layer doesnt really know what a UVM is?